### PR TITLE
ADQ-5-upgrade-data-querys-health-actuator

### PR DESCRIPTION
**Pull request by pscorer**
In addition to ADQ-5, fixed minor issue with developer script dev-app.sh.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,4 @@
 target/
 
 secrets.conf
+*.pem
```
Comment (jblefkove-va) [inline .gitignore]: Is this for the certificates from `trust-dev-certs.sh` ?

**Comment by jblefkove-va** [comment]
For future stories, even though you are forking, you should still use a branch instead of committing to master.

File: .gitignore (hunk 1 above)
Comment (pscorer) [reply .gitignore]: Yes. OK to make change?

Comment (jblefkove-va) [reply .gitignore]: :shipit:

**Pull request merged**